standard/expressions.md

```diff
@@ -1150,6 +1151,200 @@ object o = (new int[3])[1];
+    ;
+
+fragment Interpolated_Regular_String_Character
+    // Any character except " (U+0022), \\ (U+005C), { (U+007B), } (U+007D), and New_Line_Character
+    : ~["\\{}\u000D\u000A\u0085\u2028\u2029]
+    ;
+
+// interpolated verbatim string expressions
+
+interpolated_verbatim_string_expression
+    : Interpolated_Verbatim_String_Start Interpolated_Verbatim_String_Mid?
+      ('{' verbatim_interpolation '}' Interpolated_Verbatim_String_Mid?)* Interpolated_Verbatim_String_End
+    ;
+
+verbatim_interpolation
+    : expression (',' interpolation_minimum_width)? Verbatim_Interpolation_Format?
+    ;
+
+Interpolated_Verbatim_String_Start
+    : '$@"'
+    ;
+
+// the following three lexical rules are context sensitive, see details below
+Interpolated_Verbatim_String_Mid
+    : Interpolated_Verbatim_String_Element+
+    ;
+
+Verbatim_Interpolation_Format
+    : ':' Interpolated_Verbatim_String_Element+
+    ;
+
+Interpolated_Verbatim_String_End
+    : '"'
+    ;
+
+fragment Interpolated_Verbatim_String_Element
+    : Interpolated_Verbatim_String_Character
+    | Quote_Escape_Sequence
+    | Open_Brace_Escape_Sequence
+    | Close_Brace_Escape_Sequence
+    ;
+
+fragment Interpolated_Verbatim_String_Character
+    // Any character except " (U+0022), { (U+007B) and } (U+007D)
+    : ~["{}]
+    ;
+
+// lexical fragments used by both regular and verbatim interpolated strings
+
+fragment Open_Brace_Escape_Sequence
+    : '{{'
+    ;
+
+fragment Close_Brace_Escape_Sequence
+    : '}}'
+    ;
+```
+
+Six of the lexical rules defined above are *context sensitive* as follows:
+
+| Rule | Contextual Requirements |
+| :--- | :---------------------- |
+| *Interpolated_Regular_String_Mid* | Only recognised after an *Interpolated_Regular_String_Start*, between any following interpolations, and before the corresponding *Interpolated_Regular_String_End*. |
+| *Regular_Interpolation_Format* | Only recognised immediately before the closing delimiter (`}`) of a *regular_interpolation*. |
+| *Interpolated_Regular_String_End* | Only recognised after an *Interpolated_Regular_String_Start* and only if any intervening tokens are either *Interpolated_Regular_String_Mid*s or tokens that can be part of *regular_interpolation*s, including tokens for any *interpolated_regular_string_expression*s contained within such interpolations. |
+| *Interpolated_Verbatim_String_Mid* *Verbatim_Interpolation_Format* *Interpolated_Verbatim_String_End* | Recognition of these three rules follows that of the corresponding rules above with each mentioned *regular* grammar rule replaced by the corresponding *verbatim* one. |
+
+*Note:* The above rules are context sensitive as their definitions overlap with those of
+other tokens in the language. *end note*
+
+*Note:* The above grammar is not ANTLR-ready due to the context sensitive lexical rules. As with
+other lexer generators ANTLR supports context sensitive lexical rule, for example using its *lexical modes*,
```

Comment:
lexical rule => lexical rules

(unless I'm misreading)

Reply:
Thanks, will be fixed.